src/test/resources/stub/merchant/merchantDetailError.json

```
@@ -0,0 +1,22 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPathPattern": "/idpay/merchant/portal/initiatives/INITIATIVEID",
+    "headers": {
+        "x-merchant-id": {
+          "equalTo": "ERRORMERCHANTID"
+        }
+    }
+  },
+  "response": {
+    "status": "429",
```

we didn't expect a 429, a 429 should be handled with a retry, it's better to provide the generic 500
```suggestion
    "status": "500",
```